.github/workflows/application.yml

```diff
@@ -33,8 +34,8 @@ jobs:
     timeout-minutes: 5
     strategy:
       matrix:
-        java-version: [ "8", "11", "17", "20" ] # LTS + Latest
-        os: [ "ubuntu-latest", "macos-latest", "windows-latest" ]
+        java-version: ["8", "11", "17", "21", "22"] # LTS + Latest
```

Comment:
We are reaching the point where the build time is nonsensically long.

What is our user bases of each version?

Reply:
Build time is the same, they all run in parallel. You could add 100 versions here and it would take the same amount of time